draft-ietf-rats-uccs.md

```diff
@@ -475,6 +475,30 @@ an UCCS by enclosing it with a tag number TBD601:
 <!--  LocalWords:  Verifier's CWTs Attester Verifier FCFS
  -->
 
+# JSON Support
+
+The above definitions, concepts and security considerations all may be applied to define a JSON-encoded unsigned token.
+This can be referred to as a "UJCS", an Unprotected JWT Claims Set".
+The CDDL in the appendix above works for JSON as well as CBOR through the use of the JC<> CDDL generic.
+The definition for a UJCS token in CDDL is identical to that of a Claims-Set.
+
+~~~ cddl
+UJCS-Token = Claims-Set
+~~~
+
+# EAT
+
+The following CDDL adds UCCS and UJCS tokens to EAT using the predefined extension points.
```

Comment:
```suggestion
The following CDDL definition adds types for UCCS and UJCS to EAT using predefined extension points.
```

Reply:
Indeed, UCCS and UJCS are not tokens.
Can we say "its predefined"  (to make clear that it is EAT that predefines these)?

Reply:
incorporated in the suggestion above